### DISCUSSION TITLE
No changelog for gitsubmodule-created PR

### DISCUSSION THREAD

**Discussion by okainov** [renovatebot/renovate#10385] [Suggest an Idea]
**How are you running Renovate?**

- [ ] WhiteSource Renovate hosted app on github.com
- [x] Self hosted

Please select which platform you are using:

- [x] github.com

Renovate version: latest docker

**Describe the bug**
1. Configure renovate to use git-submodules
2. Have some not up-to-date submodule in your repo
3. Run Renovate
4. Check created PR
5. Make sure it doesn't contain "changelog". I expected to see list of new commits with their commit messages
![image](https://user-images.githubusercontent.com/918446/121513090-c0da6300-c9ea-11eb-914a-8a9473cf826d.png)

**Comment by rarkins** [comment]
Changelogs are displayed when:
- The source repository has a CHANGELOG file
- The update is to and from *versions* which can be mapped to CHANGELOG entries

I think in your example both are missing.

What you're probably hoping for is a dump of the commit messages? That would be quite "noisy" - a single update may have dozens or hundreds of commits, many of them with very little description.

If you want something like this then you need to do both of these:

1. Create a public reproduction repo which can be used later for implementation testing, including both the source and destination repos with one outstanding submodules PR

2. Propose this as a feature request and how you think it should work. Keep it generic instead of only git-submodules, because it needs to be considered for all update types. Things to consider include: if a repository has a CHANGELOG file already, are the individual commits worth adding.

**Reply by okainov** [reply]
>What you're probably hoping for is a dump of the commit messages? That would be quite "noisy" - a single update may have dozens or hundreds of commits, many of them with very little description.

Maybe then it can be configurable, but I think it would be straightforward and easy and, moreover, always available (in contrast with specific "changelogs")
